src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -524,6 +527,40 @@ public void after() throws Exception {
         }
     }
 
+    /**
+     * Internal method to stop Jenkins instance.
+     *
+     * @param server    server on which Jenkins is running.
+     * @param tearDowns tear down methods for tests
+     * @param jenkins   the jenkins instance
+     * @since TODO
+     */
+    public static void _stopJenkins(Server server, List<LenientRunnable> tearDowns, Jenkins jenkins) {
+        jettyLevel(Level.WARNING);
+        try {
+            server.stop();
+        } catch (Exception e) {
+            // ignore
```

Comment:
```suggestion
            LOGGER.log(Level.WARNING, null, e);
```

or better, find some way to throw it up and fail the test (using `addSuppressed` as needed).